packages/e2e/cypress/e2e/api/projectPermissions.cy.ts

```diff
@@ -0,0 +1,801 @@
+import { SEED_PROJECT } from '@lightdash/common';
+
+const apiUrl = '/api/v1';
+
+const runqueryBody = {
+    dimensions: ['customers_customer_id'],
+    metrics: [],
+    filters: {},
+    sorts: [{ fieldId: 'customers_customer_id', descending: false }],
+    limit: 500,
+    tableCalculations: [],
+    additionalMetrics: [],
+};
+const sqlQueryBody = { sql: 'select 1' };
+describe('Lightdash API tests for member user with admin project permissions', () => {
+    let email;
+    beforeEach(() => {
+        cy.loginWithPermissions('member', [
```

Comment:
big test file where we check all endpoints with different project permissions for `member` users